src/add/operator/exhaust.ts

```diff
@@ -2,4 +2,4 @@ import {Observable} from '../../Observable';
 import {exhaust} from '../../operator/exhaust';
 Observable.prototype.exhaust = exhaust;
 
-export var _void: void;
\ No newline at end of file
+export var _void: void;
```

Comment:
Is there a standard with no newlines at the end of files for the project, or was just that how things happened to end up?  (Just want to get it right! :) )